flac.md

```diff
@@ -1,371 +1,104 @@
+- **FIR Linear prediction**. For more accurate modelling (at a cost of slower encoding), FLAC supports up to 32nd order FIR linear prediction (again, for information on linear prediction, see [audiopak](http://www.hpl.hp.com/techreports/1999/HPL-1999-144.pdf) and [shorten](http://svr-www.eng.cam.ac.uk/reports/abstracts/robinson_tr156.html)). The reference encoder uses the Levinson-Durbin method for calculating the LPC coefficients from the autocorrelation coefficients, and the coefficients are quantized before computing the residual. Whereas encoders such as Shorten used a fixed quantization for the entire input, FLAC allows the quantized coefficient precision to vary from subframe to subframe. The FLAC reference encoder estimates the optimal precision to use based on the block size and dynamic range of the original signal.
 
 # Residual Coding
 
-FLAC currently defines two similar methods for the coding of the error
-signal from the prediction stage. The error signal is coded using Rice
-codes in one of two ways: 1) the encoder estimates a single Rice
-parameter based on the variance of the residual and Rice codes the
-entire residual using this parameter; 2) the residual is partitioned
-into several equal-length regions of contiguous samples, and each region
-is coded with its own Rice parameter based on the region's mean. (Note
-that the first method is a special case of the second method with one
-partition, except the Rice parameter is based on the residual variance
-instead of the mean.)
-
-The FLAC format has reserved space for other coding methods. Some
-possibilities for volunteers would be to explore better context-modelling
-of the Rice parameter, or Huffman coding. See
-[LOCO-I](http://www.hpl.hp.com/techreports/98/HPL-98-193.html) and
-[pucrunch](http://www.cs.tut.fi/~albert/Dev/pucrunch/packing.html) for
-descriptions of several universal codes.
+FLAC currently defines two similar methods for the coding of the error signal from the prediction stage. The error signal is coded using Rice codes in one of two ways:
+1. the encoder estimates a single Rice parameter based on the variance of the residual and Rice codes the entire residual using this parameter;
+1. the residual is partitioned into several equal-length regions of contiguous samples, and each region is coded with its own Rice parameter based on the region's mean. (Note that the first method is a special case of the second method with one partition, except the Rice parameter is based on the residual variance instead of the mean.)
```

Comment:
If all in 1, then markdown does the numbering starting at 1. (Has the advantage that you can insert and delete items easily.) But I can change again.